packages/app/src/app/overmind/namespaces/dashboard/actions.ts

```diff
@@ -369,7 +354,7 @@ export const getSandboxesByPath: AsyncAction<string> = async (
       path: '/' + path,
       teamId: state.activeTeam,
     });
-    if (!data || !data.me || !data.me.collection) {
+    if (typeof data?.me?.collection?.sandboxes === 'undefined') {
```

Comment:
I actually perfer the old version, find it easier to understand

Reply:
The thing is that the old version doesn't differentiate between `[]` and `undefined`, so we can't set the loading state because of this.